lcUILua/ui/commandline.lua

```diff
@@ -103,4 +103,5 @@ add_command("ROTATE", function(id) run_basic_operation(id, RotateOperation) end)
 add_command("COPY", function(id) run_basic_operation(id, CopyOperation) end)
 add_command("SCALE", function(id) run_basic_operation(id, ScaleOperation) end)
 add_command("REMOVE", function(id) run_basic_operation(id, RemoveOperation) end)
-add_command("TRIM", function(id) run_basic_operation(id, TrimOperation) end)
\ No newline at end of file
+add_command("TRIM", function(id) run_basic_operation(id, TrimOperation) end)
+add_command("LCTOLERANCE", function(id) lctolerVal(id) end)
```

Comment:
I don't like this as you have to add a new command for each parameter. I suggest to edit the command function : if the command is not found, check if it's a parameter and show its value